discovery-provider/src/queries/queries.py

```diff
@@ -1982,3 +1983,121 @@ def get_top_genre_users():
 
 
         return api_helpers.success_response({ 'user_ids': user_ids })
+
+
+
+# Get the tracks that are 'children' remixes of the requested track
+# The results are sorted by if the original artist has reposted or saved the track
+@bp.route("/remixes/<int:track_id>/children", methods=("GET",))
+def get_remix_tracks(track_id):
+
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        # Fetch the parent track to get the track's owner id
+        parent_track = session.query(Track).filter(
+            Track.is_current == True,
+            Track.track_id == track_id
+        ).first()
+        
+        if (parent_track == None):
+            return api_helpers.error_response("Invalid track_id provided", 400)
+
+        track_owner_id = parent_track.owner_id
+        # Get the 'children' remix tracks
+        # Use the track owner id to fetch reposted/saved tracks returned first 
+        base_query = session.query(Track).join(
+            Remix,
+            and_(
+                Remix.child_track_id == Track.track_id,
+                Remix.parent_track_id == track_id
+            )
+        ).outerjoin(
+            Save,
+            and_(
+                Save.save_item_id == Track.track_id,
+                Save.save_type == SaveType.track,
+                Save.is_current == True,
+                Save.is_delete == False,
+                Save.user_id == track_owner_id
+            )
+        ).outerjoin(
+            Repost,
+            and_(
+                Repost.repost_item_id == Track.track_id,
+                Repost.user_id == track_owner_id,
+                Repost.repost_type == RepostType.track,
+                Repost.is_current == True,
+                Repost.is_delete == False
+            )
+        ).filter(
+            Track.is_current == True,
+            Track.is_delete == False
```

Comment:
Should we also filter our `is_hidden`? Probably right?

Reply:
+1